superset-frontend/src/chart/Chart.jsx

```diff
@@ -29,6 +29,8 @@ import ErrorBoundary from 'src/components/ErrorBoundary';
 import { Logger, LOG_ACTIONS_RENDER_CHART } from 'src/logger/LogUtils';
 import ChartRenderer from './ChartRenderer';
 import { ChartErrorMessage } from './ChartErrorMessage';
+import { URL_PARAMS } from 'src/constants';
```

Comment:
fix these import issues

Reply:
@hughhhh I fixed these real quick so that I could use your branch for testing locally.